tracer/src/Datadog.Trace/AppSec/Waf/Encoder.cs

```diff
@@ -443,7 +443,7 @@ void EnumerateAndEncodeIList(IList lstInstance)
                 }
 
                 default:
-                    Log.Warning("Couldn't encode object of unknown type {Type}, falling back to ToString", o.GetType());
+                    Log.Debug("Couldn't encode object of unknown type {Type}, falling back to ToString", o.GetType());
```

Comment:
tiny nit, given calling `o.GetType()` could be relatively expensive
```suggestion
                    if (Log.IsEnabled(Vendors.Serilog.Events.LogEventLevel.Debug))
                    {
                        Log.Debug("Couldn't encode object of unknown type {Type}, falling back to ToString", o.GetType());
                    }
```